src/main/java/com/commercetools/project/sync/Syncer.java

```diff
@@ -100,10 +100,12 @@ public Syncer(
    * this sync.
    *
    * @param runnerName the name of the sync runner.
+   * @param isFullSync whether to run a delta sync (based on the last sync timestamp) or a full
```

Comment:
```suggestion
   * @param isFullSync whether to run delta sync (based on the last sync timestamp) or full
```

Reply:
but why? we need articles before nouns in english :)